ci/nosetests.sh

```diff
@@ -9,13 +9,17 @@ echo "nosetests -a '!requires_sudo' output"
 echo "========================================="
 nosetests -a '!requires_sudo' tests $NOSEOPTS || EXIT_CODE=$(($EXIT_CODE || $?))
 echo EXIT_CODE no_sudo nosetests = $EXIT_CODE
-sudo mv .coverage coverage.$(uuidgen).no_sudo.data
+
+[[ $NOSEOPTS == *"--with-coverage"* ]] && \
```

Comment:
should we instead check that the file exists?

Reply:
Sure, that definitely makes more sense. Will update